mix.exs

```diff
@@ -48,7 +48,7 @@ defmodule NewRelic.Mixfile do
       # Instrumentation:
       {:plug, ">= 1.10.4", optional: true},
       {:plug_cowboy, ">= 2.4.0", optional: true},
-      {:phoenix, ">= 1.5.5", optional: true},
+      {:phoenix, ">= 1.5.5 or ~> 1.6.0", optional: true},
```

Comment:
It's not clear to me why this would be needed, the existing version requirement should match:

```elixir
iex> Version.match?("1.6.0", ">= 1.5.5")
true
```

Are you encountering an error?

Reply:
You're right. My bad! It was caused by another dependency.  I'll close this PR.